KamiWize/pr-test
# feat(KAMI-234): added new settings module

## What does this PR do?
Integrates a new module.


## Where should the reviewer start?
``settings.js``


## How should this be manually tested?

1. Checkout to the current branch.
2. npm install & npm start.
3. See the logs.


## Any background context you want to provide?



## What are the relevant tickets?

[KAMI-234](https://wizeline.atlassian.net/browse/KAMI-234)


## Screenshots
<img width="1728" alt="Captura de Pantalla 2022-12-21 a la(s) 13 46 47" src="https://user-images.githubusercontent.com/110576964/214168156-6788677a-2ba5-407c-bd86-4ce28c7aa7c6.png">


## Questions :question:



#### Checklist
